board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
No, setting MODEL=blah.dtb sets the variable fdtfile into uboot env, which ubldr will import and use as the default dtb to load.
See https://svnweb.freebsd.org/ports/head/sysutils/u-boot-olimex-a20-som-evb/files/patch-include_configs_sunxi-common.h?revision=418446&view=markup#l44